cabot/cabotapp/tests/tests_basic.py

```diff
@@ -859,3 +859,29 @@ def test_get_all_duty_officers(self):
                             (officers[1][0].username, officers[1][1][0].name)]
         self.assertIn(('dolores@affirm.com', 'Principal'), officer_schedule)
         self.assertIn(('maeve@affirm.com', 'Secondary'), officer_schedule)
+
+
+class TestUserPermissions(LocalTestCase):
```

Comment:
nit: can you put this in a new file (test_permissions.py?) The original Cabot writers put everything in one file but going forward I want to split everything out by function instead of just having this file grow more